docs/data/material/components/accordion/accordion.md

```diff
@@ -75,6 +75,28 @@ The Accordion component can be controlled or uncontrolled.
 Learn more about controlled and uncontrolled components in the [React documentation](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components).
 :::
 
+### Custom heading level
+
+By default, the Accordion uses a heading element set appropriately for the information architecture of the page. If needed, you can customize the heading level.
```

Comment:
I think this part: "the Accordion uses a heading element set appropriately for the information architecture of the page" is incorrect, in the sense that only the consumer can know if the heading element/heading level is the appropriate in the context being used.

We can unify both paragraphs (the one before the code example and the one after the code example) to keep things short:

```suggestion
By default, the Accordion uses an `h3` element for the heading. You can customize the heading using the `slots.heading` prop to ensure the correct heading hierarchy in your document.
```